quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/AbstractStartCommand.java

```diff
@@ -29,6 +30,7 @@ public abstract class AbstractStartCommand extends AbstractCommand implements Ru
     public void run() {
         doBeforeRun();
         CommandLine cmd = spec.commandLine();
+        HttpPropertyMappers.validateConfig();
```

Comment:
I don't really like to have it here, but as we do not have any better solution ATM, I'm ok with these changes for now. 
However, we'd need to consider a better way how to handle these validations (even for the import/export validations) in a more abstract way. I mean, a rational question might be: "Why do the httpPropertyMappers need to be validated in the abstract start command?". 

We should leverage the `PropertyMapper.validator()` as much as possible for the individual mappers.
Or we could introduce some validation mechanism for the whole property mappers classes. I mean, having general validations for the whole option category or sth like that.

It'd be good to have sth like:
```java
interface PropertyMapperWrapper { // or some more suitable name 
    Collection<PropertyMapper<?>> getPropertyMappers();

    default Collection<PropertyMapperValidator> getValidators(){
        return Collections.emptyList();
    }

    default boolean isEnabled() {
        return true;
    }
}

interface PropertyMapperValidator{
    void validate();  
}
```

However, it's out of the scope of this PR, and a follow-up task for it should be created. 
@shawkins WDYT?
cc @vmuzikar

Reply:
@mabartos let's move this onto #27025 - I had just updated that one to say it probably wasn't worth persuing, but if you are onboard with introducing something, then let's leave it open.

Reply:
Ok, let's move it there. As I've mentioned, it's ok for me to directly use the static method for the validation for now.

Reply:
@mabartos +1. I don’t think the validations belong here either but I'm ok with it if we move it in a follow-up.